backend/billing/views.py

```
@@ -0,0 +1,709 @@
+        # Get payment record
+        try:
+            payment = Payment.objects.get(paypal_order_id=order_id, user=request.user)
+        except Payment.DoesNotExist:
+            return JsonResponse({'success': False, 'error': 'Payment not found'}, status=404)
+        
+        # Capture the order
+        access_token = get_paypal_access_token()
+        mode = getattr(settings, 'PAYPAL_MODE', 'sandbox')
+        base_url = 'https://api-m.paypal.com' if mode == 'live' else 'https://api-m.sandbox.paypal.com'
+        
+        response = requests.post(
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)